packages/fluentui/state/project.json

```diff
@@ -2,5 +2,6 @@
   "name": "state",
   "$schema": "../../../node_modules/nx/schemas/project-schema.json",
   "projectType": "library",
-  "implicitDependencies": []
+  "implicitDependencies": [],
+  "tags": ["react-northstar"]
```

Comment:
these are used to create path aliases as part of webpack build https://github.com/microsoft/fluentui/blob/master/scripts/gulp/src/config.ts#L16-L30